CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contribute To Solr
+
+Thank you for your interest in contributing to the Apache Solr project. Solr continues to improve via the many contributions of developers around the world and new contributions are welcomed and encouraged.
+
+Your suggestions, comments and comments on Doris can be made directly through the [Apache Solr JIRA](https://issues.apache.org/jira/projects/SOLR/issues).
+
+There are many ways to participate in and contribute to Solr, such as: code implementation, test implementation, documentation improvements.
+
+If you have any questions along the way, you can reach out to the community using the resources provided [here](https://solr.apache.org/community.html#mailing-lists-chat).
+
+## How To Contribute
+
+Information on how to contribute can be found [here](https://github.com/apache/solr/blob/main/dev-docs/how-to-contribute.adoc).
+
+## Contributing Documentation
+
+One very helpful area for contribution to Solr is via documentation improvements and additions to help make Solr easier to use for others. You can read more about how to modify and update Solr documentation [here](https://github.com/apache/solr/blob/main/solr/solr-ref-guide/README.adoc).
+
+## Working With Solr Source Code
+
+Information on how to work with Solr source code and the build system can be found [here](https://github.com/apache/solr/blob/main/dev-docs/solr-source-code.adoc).
+
+## IDE Support
+
+Information on IDE support can be found [here](https://github.com/apache/solr/blob/main/dev-docs/IDEs.adoc).
+
+## JVM Support
+
+Information on JVMs can be found [here](https://github.com/apache/solr/blob/main/dev-docs/jvms.adoc).
+
+## FAQs
+
+Review FAQs [here](https://github.com/apache/solr/blob/main/dev-docs/FAQ.adoc)
```

Comment:
Same comment about 'here' applies.

BTW: Is this FAQ really up to date? If not, better not link to it?